### ISSUE TITLE
Fix:  Almanac -- TypeError: must be real number, not ValueHelper

### ISSUE THREAD

**Issue by glennmckechnie** [dcapslock/weewx-responsive-skin#15]
Using WeeWX v 5.1.0

With a fresh install of this skin, I kept getting an obscure Cheetah error that was linked to turning on the **use_almanac** option in skin.conf.

weewx.cheetahgenerator was reporting...

```
Evaluation of template /home/pi/weewx-data/skins/Responsive/index.html.tmpl failed with exception '<class 'TypeError'>'
 **** Ignoring template /home/pi/weewx-data/skins/Responsive/index.html.tmpl
 **** Reason: must be real number, not ValueHelper
 ****  Traceback (most recent call last):
 ****    File "/home/pi/weewx-venv/lib/python3.11/site-packages/weewx/cheetahgenerator.py", line 334, in generate
 ****      unicode_string = compiled_template.respond()
 ****                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 ****    File "cheetah__home_pi_weewx_data_skins_Responsive_index_html_tmpl_1729911025_777892_68829.py", line 2355, in respond
 ****    File "cheetah__home_pi_weewx_data_skins_Responsive_index_html_tmpl_1729911025_777892_68829.py", line 989, in __errorCatcher132
 ****    File "<string>", line 1, in <module>
 ****  TypeError: must be real number, not ValueHelper
```
I finally isolated it to the **Altitude:** entry and the fix is as follows...

```
diff -Naur index.html.tmpl.org index.html.tmpl
--- index.html.tmpl.org 2024-10-26 14:09:02.213226805 +1100
+++ index.html.tmpl     2024-10-26 13:59:44.380761946 +1100
@@ -683,7 +683,7 @@
                     </tr>
                     <tr>
                       <td style="border-bottom: none;">Altitude:</td>
-                      <td style="border-bottom: none;">$("%.1f&deg;" % $almanac.sun.altitude)</td>
+                      <td style="border-bottom: none;">$("%.1f&deg;" % $almanac.sun.alt)</td>
                     </tr>
                     <tr>
                       <td style="border-bottom: none;">Right ascension:</td>
```

**Comment by glennmckechnie** [comment]
And edited with the corrected fix

**Comment by dcapslock** [comment]
Fixed by [8042942](https://github.com/dcapslock/weewx-responsive-skin/commit/80429428398edcfc0044ebc5036a9a801b78b91a)

**Comment by dcapslock** [comment]
Fixed by [8042942](https://github.com/dcapslock/weewx-responsive-skin/commit/80429428398edcfc0044ebc5036a9a801b78b91a)
